src/jobservice/job/impl/gc/job.go

```diff
@@ -88,6 +91,9 @@ func (gc *GarbageCollector) Run(ctx job.Context, params job.Parameters) error {
 	if err := gc.cleanCache(); err != nil {
 		return err
 	}
+	if err := ensureQuota(); err != nil {
```

Comment:
Why we re-calculate the storage usage in the garbage collection job? The GC job doesn't change the database records about the blobs at all, right?

Reply:
As harbor returns the quota at the time when the user deletes the artifact, and harbor will remove the corresponding item in table project_blob. In other words, we can believe that the project_blob reflect reality.

Here just add one insurance step after GC to ensure the correct usage.